packages/components/src/menu/src/Menu.css

```diff
@@ -88,16 +88,20 @@ IMPORTANT: The Menu component hardcoded a few CSS values to enable dynamic scrol
     background-color: var(--o-ui-bg-alias-mid-break);
 }
 
-/* OPTION | STATE | FOCUS | WITH SELECTION */
-.o-ui-menu-with-selection .o-ui-menu-item:focus:after,
-.o-ui-menu-with-selection .o-ui-menu-item-focus:after {
```

Comment:
Just wondering what are these changes addressing?

Reply:
Using a before now, for no specific reason

Reply:
Except to follow the listbox way of doing a left border.